desc/compute/data_index.py

```diff
@@ -1320,7 +1320,7 @@
     "fun": "compute_contravariant_magnetic_field",
     "dim": 1,
     "R_derivs": [[0, 0, 0], [1, 0, 0], [0, 1, 0], [0, 0, 1]],
-    "L_derivs": [[0, 0, 0], [0, 0, 1]],
+    "L_derivs": [[0, 0, 0], [0, 1, 0], [0, 0, 1]],
```

Comment:
Don't know why this would need to be changed. Will revert.

Reply:
I think I made this change. `B^theta` depends on `sqrt(g)`, `iota`, and `lambda_z`. When we have to compute `iota` from the current input, that calculation also depends on `lambda_t`. So in that case `B^theta` is a function of `lambda_t` through the `iota` dependence. It is safer and easier to just always require this extra derivative, regardless of the input mode. So do not revert this change!

Reply:
Ok I'll keep it in. Ideally we'll merge the `constrain-toroidal-current-density` branch into master soon too.